### DISCUSSION TITLE
[docs] Toast visualization `500` error

### DISCUSSION THREAD

**Discussion by JPrisk** [chakra-ui/zag#1536] [General]
# 🐛 Bug report

Hey folks, just wanted to let you know that there is a `500` error on the [toast](https://zagjs.com/components/react/toast) -> visualize logic page

## 💥 Steps to reproduce

Visit the following link

## 💻 Link to reproduction

https://state-machine-viz.vercel.app/toast

**Comment by segunadebayo** [comment]
Thanks for the heads up @JPrisk

@anubra266 can you help debug this?

**Comment by segunadebayo** [comment]
Closing — the website has been rebuilt since this was reported.
